go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.5.0
 	github.com/prometheus/client_golang v1.12.1
 	github.com/prysmaticlabs/go-bitfield v0.0.0-20210809151128-385d8c5e3fb7
+	github.com/prysmaticlabs/prysm/v2 v2.0.1
```

Comment:
I would avoid importing prysm, it is a HUGE dependency. And we do not need it actually.